src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -258,14 +268,19 @@ private JToken TransformContentCore(ErrorBuilder errors, JsonSchemaDefinition de
                     return newObject;
 
                 case JValue value:
-                    return TransformScalar(errors, schema, file, value);
+                    return TransformScalar(errors, schema, file, value, contentFallback);
 
                 default:
                     throw new NotSupportedException();
             }
         }
 
-        private JToken TransformScalar(ErrorBuilder errors, JsonSchema schema, FilePath file, JValue value)
+        private JToken TransformScalar(
+            ErrorBuilder errors,
+            JsonSchema schema,
+            FilePath file,
+            JValue value,
+            bool contentFallback)
```

Comment:
Consider pass the root schema down here because @xichan-ms is likely going to consume `rootSchema.Rules` property here in #6541 .

```suggestion
        private JToken TransformScalar(
            ErrorBuilder errors,
            JsonSchema rootSchema,
            JsonSchema schema,
            FilePath file,
            JValue value)
```

Reply:
Updated.

---
In reply to: [489233778](https://github.com/dotnet/docfx/pull/6558#discussion_r489233778) [](ancestors = 489233778)